File: guanabara-python/src/lista02/exercicio19.py
n1 = float(input("Insira a primeira nota: "))
n2 = float(input("Insira a segunda nota: "))
media = (n1 + n2)/2

if media >= 7:
    print(f"Média: {media}. Aluno aprovado.")
else:
    print(f"Média: {media}. Aluno reprovado")

File: guanabara-c/lista02/exercicio19.c
#include <stdio.h>

int main(){

    float n1, n2, media;

    printf("Insira a primeira nota: ");
    scanf("%f", &n1);
    printf("Insira a segunda nota: ");
    scanf("%f", &n2);

    media = (n1 + n2)/2;

    if(media >= 7){
        printf("Média: %.2f. Aproveitamento: Bom.", media);
    }
    else{
        printf("Média: %.2f. Aproveitamento: Ruim.", media);
    }

    return 0;
}
